### PR TITLE
@stroncium/procfs: Create typings

### PR DESCRIPTION
**Pull request by phm07**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/stroncium__procfs/stroncium__procfs-tests.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { procfs, Procfs, ProcfsError } from 'stroncium__procfs';
```
Comment (peterblazejewicz) [inline types/stroncium__procfs/stroncium__procfs-tests.ts:1]:
```suggestion
import { procfs, Procfs, ProcfsError } from '@stroncium/procfs';
```

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
you have to modify `tsconfig.json` in order to allow other packages to find this scoped module:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages

**Comment by pauhull** [comment]
@peterblazejewicz Thanks, I didn't know that. Should be fixed now

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@pauhull thanks!

**Comment by pauhull** [comment]
Ready to merge

**Pull request merged**